test/e2e/autoscaling/autoscaling_timer.go

```diff
@@ -68,9 +68,13 @@ var _ = SIGDescribe("[Feature:ClusterSizeAutoscalingScaleUp] [Slow] Autoscaling"
 			})
 
 			AfterEach(func() {
-				// Scale down back to only 'nodesNum' nodes, as expected at the start of the test.
-				framework.ExpectNoError(framework.ResizeGroup(nodeGroupName, nodesNum))
-				framework.ExpectNoError(framework.WaitForReadyNodes(f.ClientSet, nodesNum, 15*time.Minute))
+				// Attempt cleanup only if a node group was targeted for scale up.
+				// Otherwise the test was probably skipped and we'll get a gcloud error due to invalid parameters.
+				if len(nodeGroupName) > 0 {
```

Comment:
Nit picking, but I don't like checking for empty string with len > 0.

Reply:
Eh, doesn't matter, looks like both ways are considered idiomatic in go. Ignore this.